unconference/Project_taxonomy_recommendations.md

```diff
@@ -0,0 +1,577 @@
+        * Be aware of your team’s personal preferences and distribute the work based on that (project is not just code!)
+        * Link to volunteer burnout Unconference group
+* Stakeholders
+* Definition: Stakeholders can be in one or more of the following categories
+    * Users
+    * Contributors
+    * Developers 
+    * Maintainers
+    * Funders/sponsors
+    * External formal or informal media/communications/writers
+    * Employers of any of the above categories
+* Recommendations
+    * Listed elsewhere in the document
+* Engaging with Stakeholders 
+    * Definition: how the project works with one or more categories of stakeholders (as defined above) with the goal of keeping them engaged in the project
+    * Components:
+        * Onboarding
+        * Maintaining engagement
+        * Increasing engagement
+        * Planning for stakeholder turnover (when stakeholders leave, succession planning) - tied to staffing (above)
+        * Recognition and attribution 
+    * Recommendations: 
+        * Allow individuals that performed work to represent themselves if possible. If leadership presents that work, ensure that correct attribution is given.  
+        * Formal and informal reporting to funders/sponsors that include DEI aspects on the schedule that they request
+        * Be time conscious, make the best use of everyone’s time. 
+        * When scheduling, be aware of time zones putting people outside of working hours and be aware of religious holidays, including religions outside of the dominant group of the project.
+        * Have different methods of recognition for stakeholders so that there are multiple pathways individuals can be recognized for their work and so that different stakeholders can recognize it. This helps improve visibility of work, including invisible work from non-dominant groups. 
+        * Try to have stakeholders from different groups (or all stakeholders) represented on your committees and in your project leadership. 
+        * Provide opportunities beyond code contribution for stakeholders to contribute to the project that match the preferences and styles of different people. 
+        * Offer as many community resources in as many languages and as many accessible formats as possible
+        * Link to DISCOVER cookbook
+        * Link to contributor diversity tooling Unconference group
+* Mentorship 
+    * Definition: Guidance, or direction given to contributors by more established contributors or leadership in the service of having less experienced contributors have a successful experience and growth in the project.
+    * Definition 2: A relationship between two people that helps both of them grow via regular communication. Typically one is the mentor who has particular skills and experiences, and the other is the mentee, who has a particular goal for the relationship.
```

Comment:
Do we want to pick one of these 2 definitions, or merge them? having 2 in this document probably doesn't make sense.